### ISSUE TITLE
App exclusions for Media RPC

### ISSUE THREAD

**Issue by qqawsedr8** [dead8309/Kizzy#186]
**Is your feature request related to a problem? Please describe.**

Ability to disable some apps from being used by Media RPC.

**Describe the solution you'd like**

An extra menu in Media RPC where we can select apps to be excluded from it.

**Describe alternatives you've considered**

**Additional context**

**Comment by qqawsedr8** [comment]
I believe this feature would be useful because it allows users to customize their Media RPC experience based on their preferences. For example, some users may only want music apps to work with Media RPC and not others. Allowing users to selectively choose which apps can be used with Media RPC would provide greater flexibility and control over the feature.

**Comment by dead8309** [comment]
When it comes to playing audio or video content on phones, the operating systems and hardware are designed to handle one media playback at a time. This limitation applies to all mobile devices, regardless of the specific platform or model.

To address your suggestion, `media rpc` is a feature that allows access over the currently playing track, it does not override the fundamental limitation of playing multiple media streams simultaneously on a single device.

For example: if you are using `media rpc` and watching a YouTube video and try to play Spotify in the background, the audio from Spotify won't play. This is because the device's software and hardware prioritize the currently active media playback, and attempting to play another audio source would interrupt or interfere with the existing one.

To put it simply, the technology behind smartphones and similar devices is currently unable to handle concurrent playback of multiple media sources. Therefore, if you are watching a video or listening to music on one app, any attempt to initiate playback from another app will typically be unsuccessful.

So media rpc will only pick the one which is `currently playing`

**Comment by qqawsedr8** [comment]
My suggestion is not related to playing multiple media sources simultaneously. Instead, what I'm looking for is the ability to selectively choose which apps can display their activity through Media RPC.

**Comment by qqawsedr8** [comment]
To give you a concrete example of what I'm looking for, let's say I have YouTube playing media in the background. Currently, Media RPC picks up on this and displays my YouTube activity in my status. However, I don't want this to happen as I may only want my music apps to be picked up by Media RPC.

**Comment by dead8309** [comment]
> To give you a concrete example of what I'm looking for, let's say I have YouTube playing media in the background. Currently, Media RPC picks up on this and displays my YouTube activity in my status. However, I don't want this to happen as I may only want my music apps to be picked up by Media RPC.

Then why not disable rpc when you are watching YouTube

**Comment by qqawsedr8** [comment]
I understand that disabling Media RPC when watching YouTube is an option, but it would be more convenient for me to leave it on in the background. However, I don't expect you to prioritize this feature if you have more important ones to work on. I was just curious if there was a way to configure Media RPC to only display specific media apps. If not, that's okay too.

**Comment by sussycatgirl** [comment]
Shame, I was hoping this would be a feature. I've been looking for something to show what I'm listening to from my music player on Discord (it's not Spotify so there's no official integration), but I don't want YouTube, my web browser and other apps that use the media playback widget to appear in my status
